PWGHF/Tasks/taskMcValidation.cxx

```diff
@@ -73,6 +73,8 @@ struct HfTaskMcValidationGen {
   Configurable<double> yVertexMax{"yVertexMax", 100., "max. y of generated primary vertex [cm]"};
   Configurable<double> zVertexMin{"zVertexMin", -100., "min. z of generated primary vertex [cm]"};
   Configurable<double> zVertexMax{"zVertexMax", 100., "max. z of generated primary vertex [cm]"};
+  Configurable<bool> checkEventGeneratorInfo{"checkEventGeneratorInfo", false, "Enable selection of events using the subGeneratorId information"};
+  Configurable<int> eventGeneratorType{"eventGeneratorType", 4, "Which kind of events to keep (0 = MB, 4 = charm injected, 5 = beauty injected)"};
```

Comment:
Very minor, I would not use `injected` as name, rather `triggered` since we do not inject signals but we trigger on them